src/composables/usePlayground.ts

```
@@ -42,8 +62,101 @@
+    } catch (error: any) {
+      innerResponse.body = error?.message
+      innerResponse.type = 'text/plain'
+      innerResponse.status = 500
+    } finally {
+      clearTimeout(timeoutId)
+      loading.value = false
+      const end = performance.now()
+      innerResponse.time = (end - start).toFixed(2)
+
+      response.value = innerResponse
+      return innerResponse
```

Biome (1.9.4)

[error] Unsafe usage of 'return'.

'return' in 'finally' overwrites the control flow statements inside 'try' and 'catch'.

(lint/correctness/noUnsafeFinally)